extensions/title/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "title",
+  "title": "Title",
```

Comment:
A bit too generic title, `Title` doesn't really give anything, do you have any ideas?